apps/onlyoffice.sh

```diff
@@ -48,6 +48,20 @@ sudo bash onlyoffice.sh"
     exit 1
 fi
 
+# Check if apache2 evasive-mod is enabled and disable it because of compatibility issues
+if [ "$(apache2ctl -M | grep evasive)" != "" ]
+then
+    msg_box "We noticed that 'mod_evasive' is installed which is the DDOS protection for webservices. It has comptibility issues with OnlyOffice and $
+    if [[ "no" == $(ask_yes_or_no "Do you want to disable DDOS protection?")  ]]
+        then
+            sleep 0.1
+        else
+            a2dismod evasive
+	    a2dismod mod-evasive
+            apt -y install libapache2-mod-evasive
```

Comment:
Axtually to my experiemce disabling 'a2dismod evasive' is enough. But don't know if mod-evasive also doesn't work then...

Reply:
Yeah OK, if you say so. :) The module activated in the Extra Security script is called `mod-evasive`. That's why I added it.